src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+@import "../normalize/normalize.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: 7 * $cell-size + 10 * $cell-gap;
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #{$cell-color};
+    border: #{$cell-border};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s ease;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/222667931-ef6a51a0-6dba-48a4-89a1-8e5093679c3d.png)


```suggestion
    transition: transform $transitionDuration ease;
```